build/instrumentation.build.js

```diff
@@ -104,7 +104,10 @@ export function createConfig({
             'inject',
             (varname, id) =>
               // eslint-disable-next-line max-len
-              `import {createInjector} from 'vue-runtime-helpers';const injector=createInjector({});injector('${id}',{source:${varname}})`
+              `import {createInjector} from 'vue-runtime-helpers';const injector=createInjector({});injector('${id.replace(
+                /\\/g,
+                ''
```

Comment:
![liam-neeson-why (1)](https://user-images.githubusercontent.com/5759712/195565305-bace5897-b1f9-4756-bcdb-8ecc3874848a.gif)

Reply:
Because the `id` is a path, an in windows they use backslash and break things due to scaping. It is done the same way in X. I can rollback the change or document it if you prefer